manim/cli/project/templates/Graph.py

```diff
@@ -0,0 +1,16 @@
+class Graph(GraphScene):
```

Comment:
This is also the name of [another manim class](https://github.com/ManimCommunity/manim/blob/master/manim/mobject/graph.py#L152) which would get overwritten in this case. Might lead to problems if someone tries to insert a `Graph` object here.